lighty-models/lighty-gnmi-models/lighty-gnmi-yang-storage-model/src/main/yang/gnmi-yang-storage.yang

```diff
@@ -31,21 +35,44 @@ module gnmi-yang-storage {
             "gNMI Specification Section 2.6.1 The ModelData message";
     }
 
+    grouping yang-model-name-version {
+            leaf name {
+                description "Name of the yang model";
+                type string;
+            }
+            leaf version {
+                description "Version of the yang model";
+                type module-version-type;
+            }
+    }
+
     grouping yang-model {
-        leaf name {
-            description "Name of the yang model";
-            type string;
-        }
-        leaf version {
-            description "Version of the yang model";
-            type module-version-type;
-        }
+        uses yang-model-name-version;
         leaf body {
             description "Body of the yang model";
             type string;
         }
     }
 
+    grouping additional-yang-models {
+        list additional-capability {
```

Comment:
consider using leaf-list, since we do not need to identifiy specific yang model (we simply add all additional models to capability response)